google-cloud-firestore/clirr-ignored-differences.xml

```diff
@@ -252,4 +252,10 @@
     <to>*</to>
   </difference>
 
+  <!-- Recursive Delete -->
```

Comment:
I copied what I did for BulkWriter, but clirr is still unhappy. Not sure what I'm missing here...